funcx_endpoint/funcx_endpoint/endpoint/endpoint_manager.py

```diff
@@ -265,7 +265,7 @@ def daemon_launch(self, funcx_client, endpoint_uuid, endpoint_dir, keys_dir, end
         if 'endpoint_address' in self.funcx_config:
             optionals['interchange_address'] = self.funcx_config['endpoint_address']
 
-            optionals['logdir'] = endpoint_dir
+        optionals['logdir'] = endpoint_dir
```

Comment:
This `logdir` setting seems like it doesn't belong inside of this if block. If there is a reason for it being in there please let me know